src/main/resources/swagger/api-docs.yaml

```diff
@@ -2840,17 +2840,48 @@ paths:
         - Method Configurations
         - Workspaces
       operationId: listWorkspaceMethodConfigs
-      summary: List of Method Configurations in a workspace
+      summary: List method configurations in a workspace
       parameters:
         - $ref: '#/parameters/workspaceNamespaceParam'
         - $ref: '#/parameters/workspaceNameParam'
+        - in: query
+          description: Configs for all repos, not just Agora
+          name: allRepos
+          required: false
+          type: boolean
+          default: false
       responses:
         200:
           description: List of Method Configurations
         404:
           description: Workspace does not exist
         500:
           description: Internal Server Error
+      desciption: |
```

Comment:
typo: 'desciption' is missing the letter 'r' ... and makes this entire text disappear from swagger ui.